css/layout.css

```diff
@@ -24,6 +24,13 @@
     flex-wrap: wrap;
   }
 
+  #workout-mode {
+    align-items: center;
+    height: 290px;
+    margin-bottom: 245px;
+    margin-top: 50px;
+  }
+
```

Comment:
Don't use ids as CSS targets, use classes instead

Reply:
Also, use `rem` over `px` for small sizes: https://lfz-lms.herokuapp.com/code-guides/Learning-Fuze/curriculum/effective-css